pkg/workers/push/push.go

```diff
@@ -48,8 +48,6 @@ type Message struct {
 	NotificationID string `json:"notification_id"`
 	Source         string `json:"source"`
 	ClientID       string `json:"client_id,omitempty"`
```

Comment:
Why do you still have a `ClientID` here? Is it just for retro-compatibility? If yes, you should add a comment to say so

Reply:
No we could get rid of it actually, and it is not used for the moment. I'll remove it as you suggest since I don't think we'll ever need it.